docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -998,6 +998,48 @@ public class ProductEndpoint {
 <1> The `getProduct` callback method can only be invoked if the current security identity has an `admin` role or the user is allowed to get the product detail.
 <2> The error handler is invoked in case of the authorization failure.
 
+==== Bearer token authentication
+
+The xref:security-oidc-bearer-token-authentication.adoc[OIDC Bearer token authentication] expects that the bearer token is passed in the `Authorization` header during the initial HTTP handshake.
+Some WebSocket clients like the https://vertx.io/docs/vertx-core/java/#_websockets_on_the_client[Vert.x WebSocketClient] support configuring headers to the WebSocket opening handshake, but clients that strictly follow the https://websockets.spec.whatwg.org/#the-websocket-interface[WebSockets API] do not support headers.
+The absence of headers proves a challenge when writing secure WebSocket client applications for a web browser with JavaScript.
```

Comment:
```suggestion
Therefore, passing a bearer access token using a custom `Authorization` header is impossible with JavaScript based WebSocket clients.
```
It is only a problem for passing the extra content, the suggested mitigations to make the upgrade secure are listed below